### PR TITLE
introduce links to the helm chart

### PR DESCRIPTION
**Pull request by lmsurpre**
I also expand the top-content to focus on getting started materials

Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,42 @@ The IBM® FHIR® Server is a modular Java implementation of version 4 of the [HL
 
 For a detailed description of FHIR conformance, see https://ibm.github.io/FHIR/Conformance.
 
-The server can be packaged as a set of jar files, a web application archive (war), an application installer, or a Docker image.
+The server is available in the following forms:
+* a web application archive (war)
+* a zip file with installation script
+* a [Linux container image](https://hub.docker.com/r/ibmcom/ibm-fhir-schematool) from the ibmcom org on DockerHub
```
Comment (prb112) [inline README.md:9]: schema tool is intentional?

Comment (lmsurpre) [reply README.md:9]: oops, no, good catch

Comment (lmsurpre) [reply README.md:9]:
```suggestion
* a [Linux container image](https://hub.docker.com/r/ibmcom/ibm-fhir-server) from the ibmcom org on DockerHub
```

**Comment by d0roppe** [comment]
LGTM

**Review by tbieste — APPROVED** [review]
I like the idea of linking to the helm chart in the README.

File: README.md [hunk 2]
```diff
@@ -3,17 +3,42 @@ The IBM® FHIR® Server is a modular Java implementation of version 4 of the [HL
...
+* a [Linux container image](https://hub.docker.com/r/ibmcom/ibm-fhir-server) from the ibmcom org on DockerHub
+* a [helm chart](https://artifacthub.io/packages/helm/alvearie/ibm-fhir-server) from the alvearie org on ArtifactHub
 
 ### Running the IBM FHIR Server
-The IBM FHIR Server is available from the [Releases tab](https://github.com/IBM/FHIR/releases) as a zip file with installation scripts for Mac/Linux and Windows or as a docker image at https://hub.docker.com/r/ibmcom/ibm-fhir-server.
+Guides for configuring, operating, and extending the IBM FHIR Server are available from https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide.
+
+#### From the zip installer
+Download the fhir-persistence-schema and fhir-install assets from the [Releases tab](https://github.com/IBM/FHIR/releases) and follow the instructions from the [User's Guide](https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide#21-installing-a-new-server) to:
+1. use fhir-persistence-schema-VERSION-cli.jar to deploy the schema
```
[the remark this answers is not in the document]
Comment (lmsurpre) [reply README.md]: done

**Pull request merged**